### ISSUE TITLE
Break "After you apply" content into separate pages

### ISSUE THREAD

**Issue by LMElston** [gc-proto/covid-19-guidance#114]
Per testing results (task failure/page feedback) and consistency of navigation with other benefits.

Create prototype top-level pages for:
- Keep getting your payments
- Change or cancel an application
- Return a payment
- [to-be-titled reporting/compliance page]

**Comment by LMElston** [comment]
Work in progress pages will be linked from this draft version of the landing page:
http://test.canada.ca/covid-19-guidance/proto/wage-subsidy2/index-ext.html

**Comment by LMElston** [comment]
Very tentative top-level menu/heading structure with new pages (for our discussion/input):

What the changes are
Who can apply
Periods you can apply for
Which of your employees qualify
Calculate your subsidy amount
How to apply
Keep getting your payments
- Apply again for later claim periods, When and how we send your payment, Review of claim information
Change your application
- Why you may want to change or cancel your application, How to change or cancel a claim you submitted
Return a payment
- Why you may need to return a payment, Return an entire payment, Return part of a payment
Reporting and record-keeping (not committed to this H1)
- Include the subsidy on your returns, Keep your records, Consequences of fraudulent claims
Contact us about CEWS

**Comment by lbelmore-gc** [comment]
@LMElston - thanks for getting this together. It's a great starting point.

I know Chris and Alex are away this week. There is no rush or hard deadline for these changes - but worth a discussion.

@cfarquharson / @Christopher-O / @Alireme / @qusai-barakat - have a look at Laura May's draft version on the test site as well as her notes and questions above and please add any comments to feedback

**Comment by qusai-barakat** [comment]
Hi team,

I'm coming into this a little blind in comparison to the rest so apologies if some comments were discussed in the past.

Is it possible to compile **Change your application** and **Return a payment** together? They're both pretty long, but if we were to drop some of the elements into accordions like the **If you were paid by direct deposit** and **If you were paid by cheque** sections of the **Return an entire payment** it'd reduce the amount of scrolling.

Also, is having the **Record-keeping and review** page be separated from the subway menu an option? Dropping it into a secondary side menu similar to Related guides or Statistics? 11 items in a list that's supposed to be a step-by-step for the users seems daunting and that page doesn't seem to be a necessary for the process.

**Comment by LMElston** [comment]
Thanks for your thoughts!

> Is it possible to compile **Change your application** and **Return a payment** together? They're both pretty long, but if we were to drop some of the elements into accordions like the **If you were paid by direct deposit** and **If you were paid by cheque** sections of the **Return an entire payment** it'd reduce the amount of scrolling.

Maybe? My only issue with combining these is changing the H1 to something that will let people know they can find information on returning a payment and changing an application on the same page. Some people (maybe most? I'm not sure about that part) will need to return money because CRA rejected or adjusted a claim (rather than it being because they decided to change it). But I think you're right that these two things naturally kind of fit together.

> Also, is having the **Record-keeping and review** page be separated from the subway menu an option? Dropping it into a secondary side menu similar to Related guides or Statistics? 11 items in a list that's supposed to be a step-by-step for the users seems daunting and that page doesn't seem to be a necessary for the process.

Yeah I would like to reduce the top-level items but I'm not sure what the best placement for this content is yet. How to report CEWS on your return is likely going to become more popuplar as we head into the end of the calendar year (though hopefully it will also be addressed on the T2 pages where people may be looking first? I'll try to find out), and fraud/verification is something that we seem to alternately want to promote and make less prominent...

**Comment by qusai-barakat** [comment]
Just following up on our meeting earlier today. Recommendations based on the PPT Alex presented can be found below.

Task 1 and 5 have already been addressed. Task 2 is solved with the new page divisions that Laura May has in the draft of the test site. Task 3 is a small copy change that may help with some confusion. Task 4 is the only outstanding issue.

Once we receive the missing content, we can go into the re-test in TryMyUI but I don't see it being a huge priority since most, if not all major issues have already been addressed and most new tasks there are reconfirming the changes.

Breakdown of recommendations based on tasks from the After you apply presentation:

- **Task 1:** Issue resolved with the new Periods you can apply for page.
- **Task 2:** The main issue with the After you apply page seemed to be with findability based on the test results. Splitting it up to give the different pages a relavent title like Laura May did in the prototype will likely solve that issue.
- **Task 3:** There was a recommendation for the When and how we send your payment to change it from 3 - 5 business days to 3 to 8 days (3 to 5 business days) to solve some user struggles.
- **Task 4:** Recommendation was to give users a method to check the status of their application and clarify on the After you apply page how to do it. Since this page is now split up, check the status doesn't seem to fit under either H1 of Change your application or Return a payment. If Change your application header is a little more broad it can likely go there.
- **Task 5:** Issue resolved with the new Periods you can apply for page.

**Comment by lbelmore-gc** [comment]
Thanks @qusai-barakat 

For **task 3 and 4** (timing for payments and status of application) - We discussed getting more information from the call center to confirm what they are communicating to users. I believe that Alex was contacting Roger on this. Do you know if she reached out before she left?

For **task 2** (separating the After you apply content) – thanks for confirming the results. This breakdown will address findability. The outstanding question is will these additional pages make the menu too long and visually cluttered? Is it better and still increase findability of the answers using the subpages (as seen in Chad’s example)?
http://test.canada.ca/covid-19-guidance/proto/wage-subsidy/after.html 

http://test.canada.ca/covid-19-guidance/proto/wage-subsidy2/index-ext.html 
@Christopher-O @cfarquharson @Borges-Porteous - thoughts?

**Comment by Borges-Porteous** [comment]
This is really hard to comment on Lisa!
1. Qusai's notes state that for Task 2 specifically, there was indeed a findability issue with the label "After you apply". 
If that's what the evidence says, it's hard to push Chad's model, even though the After you apply heading seems to be good for tasks 1, 3-5 and is much cleaner for the other top tasks...

I would like to recommend using Chad's cleaner and more scannable nav, but only if we can find a proper findability treatment for whatever was going on in Task 2. 

I'll let Qusai and Laura May speak to whether that's possible and what exactly Task 2 was about, and whether it was a top task for CEWS or not (because if it's not, that should also deprioritise it somewhat)

**Change your application**
Might be worth noting that "Change your application" is a top task that doesn't fully map to After you apply and that we also want to nudge people towards doing in CEWS.

Removing this from first level visibility could be an issue. Might want to test this one further. There are probably some ways to address this, such as including rescue links on the How to apply page.

**Chad's prototype is out-of-date.**
More of a housecleaning detail, but Chad's prototype is also deceptive, since it's missing some labels like "Periods you can apply for" and "What the changes are" that would bulk it out a bit. We should also respect the latest headings that Laura May has been working on like "Change your application", "Return a payment", "Recordkeeping and reporting". This all just fell a bit out of date with the double prototypes though, nothing really for or against either approach.

Tell me if I can pitch in further, but it seems like all the task content is in a good place, and this is simply a nav decision and Task 2 issue that's needs to be determined.

**Comment by qusai-barakat** [comment]
> For **task 3 and 4** (timing for payments and status of application) - We discussed getting more information from the call center to confirm what they are communicating to users. I believe that Alex was contacting Roger on this. Do you know if she reached out before she left?

I never heard anything about her reaching out. I sent him an email and will follow up when I hear back in regards to 3 and 4.

As for task 2, users didn't have an issue with the H1 label for After you apply since that's where they initially assumed they needed to be. The issue was, once they were on that page, there was too much content and the time on task was averaging 3 minutes. So, either Laura May or Chads approach will work since After you apply spoke to most users from the get go but a clearer H1 also wouldn't hurt.

**Comment by LMElston** [comment]
Re. task 2 and testing: would a basic A/B test before launch be reasonable under circumstances like this, rather than making a change and hoping we guessed correctly? I mean--we can also do that :) but since we're basically considering two nav options maybe this could be worthwhile?

**Comment by qusai-barakat** [comment]
That was my initial thought as well (re. A/B testing). I reached out to some team members on the UX team about that a couple of hours ago and they mentioned we currently don't have any tools for A/B testing. I'm unsure if tools are shared between teams yet as I've only been with the CRA (and the government as a whole) for a month now so if that's an option then that's great. Otherwise, a workaround could be to draft up two tests with the navigation options in TryMyUI and compare the results that way. Unfortunately, with that approach we'd likely only have 10 max per bag to compare and A/B testing works better with a larger sample size.

**Comment by Borges-Porteous** [comment]
If Task 2 is fine with that After you apply label, then there's no reason not to go with simplified nav. 

As Qusai says, we'll retest afterwards, which covers any unexpected issues.

@qusai-barakat Yeah, from what I heard IRCC is working with Service Canada to set up A/B testing with the toolset over there, but apparently they're not sharing access yet... CRA doesn't have an A/B testing solution of their own either though I think there might be plans for one in the drupal project for next year...

**Comment by LMElston** [comment]
Ah interesting @qusai-barakat, thanks for that info.

I'll work on getting Chad's prototype version updated with the current menu and new sub-pages/doormat text so we can see what that looks like.

**Comment by qusai-barakat** [comment]
re **task 3 and 4**, Roger will get back to me Tuesday or Wednesday next week. Will follow up as soon as I hear anything.

**Comment by lbelmore-gc** [comment]
Thanks all!! 

Glad we talked through this. We will see how things look with Laura May's updates to the subpage version and go from there.

**Comment by Borges-Porteous** [comment]
Maybe something like "Keep records as CRA reviews your file"? Or something to that effect? Then we have both an action and alert that it's CRA that has things to do.

**Comment by LMElston** [comment]
@Borges-Porteous if we use that H1, it still obvious enough that that's were you find information about how to report CEWS on your taxes?

**Comment by Borges-Porteous** [comment]
> @Borges-Porteous if we use that H1, it still obvious enough that that's were you find information about how to report CEWS on your taxes?

Nope, that's another aspect, isn't it.

Maybe "Keep records for tax filing and audits"?

**Comment by Christopher-O** [comment]
> > However I don't know if reporting/review should be under after you apply, as that is a process you do at the end of all the periods you applied for (at the end of the tax year), not after each period. The other items (return a payment/cancel an application) refers to individual application periods.
> 
> I see what you're saying... however, reporting/review does still happen after you apply--it's just not linear. It's still possible to apply for another period after you've reported the CEWS on your taxes for example (especially now that it's being extended to next summer). And you could have to return multiple payments after learning that you were never an eligible employer months down the road, for example.

Perhaps after you apply should be more specific to the period, like "after you apply for a claim period".  the reason I suggest this is because except for the keeping of records for review, all of the sub pages may not apply to everyone. An application may only apply for one period (keep gettting your payments), you may never have to change your application or return a payment.  The only one that is critical (required) for ALL applications is the record-keeping one...which then could risk seeming optional too being placed in the same sub-section.

**Comment by LMElston** [comment]
> Maybe "Keep records for tax filing and audits"?

"Audit" is too specific for this use, so I'd maybe rephrase this version to "Keep records for tax filing and review"

I like that this is task phrasing. On the down side, it still places the emphasis on "keep records" which is the least valuable information on the page (tax filing #1, verification/review #23

**Comment by Christopher-O** [comment]
> > Maybe "Keep records for tax filing and audits"?
> 
> "Audit" is too specific for this use, so I'd maybe rephrase this version to "Keep records for tax filing and review"
> 
> I like that this is task phrasing. On the down side, it still places the emphasis on "keep records" which is the least valuable information on the page (tax filing #1, verification/review #2)

I don't see it much of an issue.  Keep records is the most important part because when you have to file, or if you do get reviewed it's very valuable to have that information at hand.  So keep records is the action the applicant must do.

**Comment by LMElston** [comment]
> I don't see it much of an issue. Keep records is the most important part because when you have to file, or if you do get reviewed it's very valuable to have that information at hand. So keep records is the action the applicant must do.

I think the primary task on this page is "Report CEWS on your tax return". The questions people are going to have as we move toward filing season are going to be about whether or not CEWS is taxable, what T4 codes to use, etc. So keeping records is important (and is a task) but this H1 still sort of buries the lead to me? In addition, we offer 0 information about what specific records you need to keep--we only say you have to keep records and then link to very general information.

That said, keeping records is the tie that binds filing and validation so I do like the H1 for that reason.

**Comment by Borges-Porteous** [comment]
"Report CEWS on your tax return" is very nice! That also sounds like the important task here.

Keep records is something they need to be reminded of that when they have the opportunity to print things out. After they finished going through the calculator and application, then it's already a bit too late anyway.

As for reviewing... that's not a user task at all. It should rather be a section on the page that says that when they file their taxes, they better make sure everything was correct. If they are not sure of this then they should reapply or contact CRA to figure out their options.

**Comment by LMElston** [comment]
> As for CRA reviewing their file... that's not a user task at all. It should rather be a section on the page that says that when they file their taxes, they better make sure everything was correct. If they are not sure of this then they should reapply or contact CRA to figure out their options. Almost like a bit of a rescue at the bottom of the page? So a good H2 there is probably enough, and should stand out in the on this page.

It's beginning to sound like this content does not belong on the same page.

The information about review involves a few different stages of CEWS--primarily, it relates to what to expect if CRA calls you or sends you a letter to verify information you entered on your application. When this happens, it is always after you apply, but sometimes before we send the payment (for example, applicants may get calls if we are checking their address/bank account).

This content also discusses the more formal verification that may happen down the road where we ask you to prove your information by showing your receipts, so to speak. This can happen before or after you file your taxes. It also has the information about the consequences of fraud, how to report leads, etc.

It's more topic than task information...

**Comment by Borges-Porteous** [comment]
A separate page could well be it. "Pay penalties for over-claims" perhaps, where people can read that to find out how and when over-claims are determined and what they'll need to do to prove they aren't overclaiming, etc. 

It depends whether they are tied together by a "Keep records" task, or if taxes is the task people are actually interested in, meaning that "Prepare for a CRA review" ends up standing separately.

**Comment by LMElston** [comment]
As food for thought (while I go attend to more pressing P8 matters for a moment), here are a few reasons someone might want the review content...
- CRA called them to verify their CEWS claim and they don't know whether or not it's a scam
- someone is thinking about fudging the numbers and wants to know how bad it will be for them if they're wrong
- an applicant wants to be prepared in case they get asked to prove their claim at some point
- someone is wondering if anyone is actually checking up on all the "free money" the government is sending out to businesses
- someone knows their employer/whoever is lying on their application and wants to know what to do about it

**Comment by Christopher-O** [comment]
I agree with @Borges-Porteous that Report CEWS on your tax return is a good title. 
As for the other points they are pretty independent of each other.  If we are breaking up after you apply, then that is the fall out of these types of situations, it makes it difficult to re-categorize them easily.

**Comment by LMElston** [comment]
I've updated Chad's mock-up with a possible version of the sub-pages. I haven't done the right menu on all the sub-pages yet (or polished up the content) but in case you want to take a look: 

http://test.canada.ca/covid-19-guidance/proto/wage-subsidy2/cews-after-apply.html

@Christopher-O should the right menu be expanded to show the sub-pages on this type of page when you're on the parent page, or only when you're on a child page?

**Comment by Christopher-O** [comment]
@LMElston it only needs to be visible on the child pages as the main page (after you apply) already has the links to the child pages, so its basic duplication of links which could be confusing.

**Comment by qusai-barakat** [comment]
Hi team,

I heard back from Roger in regards to tasks 3 and 4

**Task 3:**

Quote from Roger:
"For the 3-5 Business days that is normally how we word it however, adding excluding weekends and holidays may clarify it."

Two recommended options here. The second option was recommended by Alex in the After you apply findings presentation. The 3 - 8 days includes both, weekends as well as holidays if it's relevant:
**- 3 to 5 business days (excluding weekends and holidays)
- 3 - 8 days (3 - 5 business days)**

**Task 4:**

Quote from Roger:
"They can see some information in My Business account but not all statuses, our agents can see more information in the source systems."

Since users can see _some_ information on their My Business account, content around that should likely be on the Keep getting your payments subpage (though this H1 doesn't seem completely relevant to checking the status). A suggestion for content to add:
"To check the status of your application, please check your My Business account."

We likely won't want to encourage the users to contact the call centre right away so having that line in there will ensure the majority of users will likely get the information needed from their My Business account page, and the special cases would then need to contact the call centre for more information.

Thoughts?

**Comment by LMElston** [comment]
Thanks @qusai-barakat, that's informative.

My preference for task 3 is to say "3 to 5 business days (excluding weekends and holidays)". I think it's clearer.

For task 4, I like your sentence. I might change it a little to "To check the status of your application, please go to My Business Account" (just to remove the double 'check'). Then, I think you're right, if people still don't see the status they will be motivated to figure out how to call us.

I guess Roger didn't say anything about how long they want people to wait before calling to check on the status?

"Keep getting your payments" is not my favourite H1 anyway (though I don't have an alternate to propose just at the moment), how tied are we to keeping it parallel with the other C-benefits?

**Comment by Christopher-O** [comment]
@qusai-barakat @LMElston correct me if I'm wrong but I thought from Alex's presentation that it was the phrase "business days" that was tripping people up, not that it wasn't "clearer".  Its kind of odd to say 3 to 5 business days and then say excluding weekends and holidays as per our (CRA) schedule, weekend and holidays are never a business day so they can never be excluded.  It's like saying I eat all types of pasta excluding steak and pork chops (they are food, but not pasta).  

I think it was further exampled that not every business is open business days (Monday to Friday) hence the confusion.  So if that's the logic, if a business was only open Friday, Saturday and Sunday and they consider that "their" business days....would that then mean that technically they have to wait 3 to 5 Fridays (thus 3 to 5 weeks) to get their payment because that's the only business day that is NOT a weekend or a holiday to them.

**Comment by qusai-barakat** [comment]
@Christopher-O Exact recommendation from Alex is as follows:

**Issue:** People knew where to find the information but struggled to connect “3-5 business days” with actual days of the week.
**Recommendation:** “Business days” is where people struggled most. Counting weekends in our estimates will provide a clearer sense of time for applicants.

Business days was what was confusing people, which is why both recommendations take weekends into account. And you're technically right. If a business is only open Fri - Sun then they may assume it means waiting 3 - 5 weeks. I think if we want to avoid any and all confusion we shouldn't use "business days" at all and alter Alex's recommendation to just say **3 - 8 days** rather than **3 - 8 days (3 - 5 business days)** ? If someone then receives their payments at the lower end of that 3 - 8 days (if they applied on a Monday for example) then they'll just think that the CRA is quick!

What do you think?

**Comment by Christopher-O** [comment]
@qusai-barakat  I thought the original recommendation was to change it to: 

3 to 8 days (excluding weekend or holidays) 

Then the phrase "business days" are not even a part of the lexicon.  If someone is knowledgeable about what a business day (per the common use of the term) is, then they know it's not a weekend or a holiday already.  Is there a best practice on keeping that phrase in?  If not, why not drop it completely.

**Comment by qusai-barakat** [comment]
@Christopher-O here's the recommendation:
![image](https://user-images.githubusercontent.com/71138731/96190783-c4c3ed00-0f10-11eb-8f9b-8a9043cb2e1a.png)

Correct me if i'm wrong but that's the recommendation I saw in the PPT document. Which is why I was mentioning maybe just dropping the bracket text completely may be the best solution. 3 - 8 days is simple and straight to the point.

**Comment by Christopher-O** [comment]
@qusai-barakat  I agree.  Unless there is some fundamental reason to keep the phrase "business days", I'm all for dropping it.

**Comment by qusai-barakat** [comment]
I'll let others speak to that. There may have been discussions in the past about keeping that phrase in that I'm not aware of.

**Comment by Alireme** [comment]
@Christopher-O @qusai-barakat My understanding was that there had previously been a lot of pushback for taking out "business days" on some of the other products, possible due to leg? Unclear. So it was kept in the brackets to satisfy that. If we are able to take that out and there's no formal reason for including "business days", I'm all for removing it. 

Otherwise, @qusai-barakat was pretty spot on with most comments!

**Comment by LMElston** [comment]
"Business days" might be hard to remove because CRA uses that term everywhere--I've floated that particular change (3 - 8 days) to ABSB to see what they think about it and will let you know. I thought 'weekdays' might be an acceptable alternative but if people don't understand business days maybe they also don't understand how 3 weekdays wouldn't include a weekend? I don't know. This bit of user evidence is throwing me a bit tbh! It is not something I thought people would have had an issue with. Just another reason we test :)

In the meantime, let's see if we can nail down a structure we like and then I/we can assess how much content might need to change based on any new sections/pages. That will help us know if this would be manageable to include with the upcoming launch.

**Comment by LMElston** [comment]
Just a follow-up on checking the status of an application (**task 4**). I tried to slip in some wording around going to MyBA to check the status of your application during a related revision to the existing "After you apply" page from ABSB. Unfortunately, business enquiries came back and said the only thing applicants can see in MyBA is whether or not they have a CEWS balance. So if we want to add text about checking the status, we'll have to do a little digging on specifically what they can see there and/or how best to advise people to check the status of their application.
